html-forms/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="#">
+    <label for="email">Email <br />
+      <input type="text" name="email" id="email" />
+    </label>
+    <div>Why is coding cool?</div>
+    <select name="answer">
+      <option value="option">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
```

Comment:
You need a `label` for this `select`. I would recommend turning the `div` on line 15 into a `label`.